src/magql/resolver_factory.py

```diff
@@ -483,8 +483,6 @@ def mutate(self, instance, parent, info, *args, **kwargs):
         info
         :return: the value that was deleted
         """
-        session = info.context
-        session.delete(instance)
```

Comment:
Does it make more sense to remove it from `post_resolve`? @Tworley96 the docs are the same for `mutate` and `post_resolve`, does that need to be fixed?